modules/oadp-deleting-backups.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-deleting-backups_{context}"]
+= Deleting backups
+
+You can remove backup files by deleting the `Backup` custom resource (CR).
+
+[WARNING]
+====
+After you delete the `Backup` CR and the associated object storage data, you cannot recover the deleted data. 
+====
+
+.Prerequisites
+
+* You created a `Backup` CR.
+* You know the name of the `Backup` CR and the associated namespace. 
+
+.Procedure
+
+* Choose one of the following actions to delete the `Backup` CR:
+
+** To delete the `Backup` CR and keep the associated object storage data, issue the following command:
++
+[source,terminal]
+----
+$ oc delete backup <backup_CR_name> -n <velero_namespace>
+----
+
+** To delete the `Backup` CR and the associated object storage data, issue the following command:
++
+[source,terminal]
+----
+$ velero backup delete <backup_CR_name> -n <velero_namespace>
```

Comment:
Do we need to add an extra prerequisite to make sure the customer has the `velero` binary installed/configured? I didn't see this prerequisite elsewhere on the assembly where this module is included but I'm not an expert in this area.

Reply:
Added two prereqs (taken from Troubleshooting section) and added an xref. 

:heavy_check_mark: